src/bun.zig

```diff
@@ -838,6 +839,41 @@ else if (Environment.isWindows)
 else
     std.os.fd_t;
 
+pub const FileDescriptor = enum(FileDescriptorInt) {
```

Comment:
enums are also ints in codegen and I can consolidate them later into one type in a followup but decided it would be better to hold off and not do it in this one